src/justbases/_errors.py

```diff
@@ -45,11 +45,8 @@ def __init__(self, op, other=None):
 
     def __str__(self):
         if self._other is None:
-            return "invalid operation for Radix: %s" % self._operator
-        return "invalid operation %s for Radix and %s" % (
-            self._operator,
-            type(self._other).__name__,
-        )
+            return f"invalid operation for Radix: {self._operator}"
+        return f"invalid operation %s for Radix and {(self._operator,type(self._other).__name__,)}"
```

Comment:
This transformation is not quite correct.

Reply:
OK; I have a feeling that there might be an issue with the second `return` statement; I spotted a comma at the end that I probably don't need (but isn't causing a failure in pylint).

Reply:
I think, generally speaking, there should be no "%s" in the f-string.

Reply:
I think I found out what was missing.